Disable in specific worlds

stan60250 opened the pull request:
Added language Chinese(traditional) and disable in specific worlds feature.

stan60250 committed:
Added language Chinese(traditional) and disable in specific worlds feature

ljacqu commented:
-1 

This repeats the same line of code over and over -> should be its own method

connection-lost commented:
> Its own method

The `isLockable(Block block)` and `isLocked(Block block)` method in LocketteProAPI.java to the rescue. We only need to add the world check code twice.

stan60250 committed:
Update config method

stan60250 committed:
Update LocketteProAPI and remove duplicate code

connection-lost commented:
One more thing:
What's with those "ignoreCancelled" you just added? Was that intentional?

stan60250 commented:
Yes. I replaced `if (event.isCancelled()) return;` with `@EventHandler(ignoreCancelled = true)`
It's same but is it much more easier?
[https://bukkit.org/threads/ignore-cancelled-events.61155/](url)
[https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/event/EventHandler.html](url)

connection-lost commented:
Ah I see.

Pawned committed:
Ignore case-sensitivity on Clan Tags

Clan Tags are not case-sensitive (i.e. 1 user cannot have clan tag "Test" and another user have a clan with tag "TeSt")
This update just makes it easier to give their clan members access as we found that .getTag() returns the tag in all lowercase while users were used to typing their tag with whatever casing they usually have on their tag.

cyilin committed:
prevent mobs break block. fix #75

HSGamer committed:
Merge pull request #1 from stan60250/master

Merge

HSGamer committed:
Merge pull request #2 from Pawned/patch-1

Ignore case-sensitivity on Clan Tags

HSGamer committed:
Merge pull request #3 from cyilin/master

prevent mobs break block. fix #75

stan60250 committed:
Added exhibit(show-off) feature
